### ISSUE TITLE
Question about center frame

### ISSUE THREAD

**Issue by Brian417-cup** [Vegetebird/MHFormer#106]
This is a wonderful job for 3D pose estimation! Here, I have a question about the "center frame" you have shown in the code. I discovered in the training phase, the output shape of model is [N,T>1,V,C=3], and you directly calculate the loss with a ground truth 3D label that has the same shape [N,T>1,V,C=3]. And in the evaluation phase, you choose the center frame and calculate error loss with one single frame ground truth 3D label. So, can I understand the idea of "center frame" just being utilized in the evaluation phase rather than training phase that ensure the final result is more accurate?

**Comment by Vegetebird** [comment]
Yes, you are right. We can also evaluate the results using the full frame, and the performance is similar.
